account-kit/react/src/components/auth/card/error/icons/wallet-icon.tsx

```diff
@@ -12,18 +9,7 @@ export const WalletIcon = ({ walletType }: { walletType: EOAWallets }) => {
   if (availableWallets.includes(walletType)) {
```

Comment:
oh same comment here, I dunno if this is the right thing to be checking. we support ALL injected wallets not just MM and Coinbase